doc/fuzzing.md

```diff
@@ -51,6 +57,13 @@ export AFL_HARDEN=1
 cd src/
 make
 ```
+
+If you are using clang you will need to substitute `afl-gcc` with `afl-clang`
+and `afl-g++` with `afl-clang++`, so the first line above becomes:
+```
+./configure --disable-ccache --disable-shared --enable-tests --enable-fuzz CC=${AFLPATH}/afl-clang CXX=${AFLPATH}/afl-clang++
```

Comment:
Maybe use `afl-clang-fast` here? It's a bit weird that this is recommended below. Unless there's some downside.

Reply:
My takeaway from the [docs here](https://github.com/mirrorer/afl/tree/master/llvm_mode) is that `afl-clang-fast` is not as stable as `afl-clang` and that it will become `afl-clang` once it has become stable enough, at which point `afl-clang-fast` would probably be removed. So I thought keeping `afl-clang` would be the better option to keep as the default.

Reply:
I see. Might be useful to mention that potential downside below.